src/veins/base/toolbox/Signal.cc

```diff
@@ -577,415 +359,158 @@ Signal& Signal::operator+=(const Signal& other)
 {
     assert(this->getSpectrum() == other.getSpectrum());
```

Comment:
I feel like Signal operators that take another Signal should make sure that either this or the other Signal has `timingUsed==false`.

Reply:
Agreed so far. They should either have no timing at all (which I would deprecate in the future anyhow) or have the same timing.